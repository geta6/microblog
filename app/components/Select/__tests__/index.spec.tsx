import React from 'react';
import renderer from 'react-test-renderer';
import { SelectComponent } from '..';

describe('SelectComponent', () => {
  test('snap shot', () => {
    const component = renderer.create(<SelectComponent />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
