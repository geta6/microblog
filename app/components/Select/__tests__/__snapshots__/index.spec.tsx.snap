// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectComponent snap shot 1`] = `
<div
  className="style__Container-yir4sn-0 ekeCgB"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  <div
    className="style__Contents-yir4sn-1 hQOxkZ"
    onClick={[Function]}
  >
    PINK
  </div>
  <div
    className="style__Contents-yir4sn-1 hQOxkZ"
    onClick={[Function]}
  >
    CYAN
  </div>
</div>
`;
